playbook/approval.go

```diff
@@ -52,6 +52,11 @@ func requiresApproval(spec v1.PlaybookSpec) bool {
 
 func approveRun(ctx context.Context, run *models.PlaybookRun) error {
 	approver := ctx.User()
+	if abacResource, err := runToABACResource(ctx, *run); err != nil {
+		return ctx.Oops().Wrap(err)
+	} else if !rbac.HasPermission(ctx, approver.ID.String(), abacResource, rbac.ActionPlaybookApprove) {
```

Comment:
i think we should add `GetPermissionObjects() []Object` to  a models/accessors.go 

for a playbook run  initially we just return the playbooks, but in subsequent phases we would return the objects referred to as well - the permission check than applies to all.